### PR TITLE
Add additional configuration options for the SSE server

### LINKED ISSUE #57: Add additional configuration options
There are a few settings users will probably want to tweak for performance or security reasons:
- Access-Control headers to control [allowed origins](https://developer.mozilla.org/en-US/docs/Web/HTTP/Server-Side_Access_Control)
- Whitelist/blacklist for queues clients can access, otherwise they'll be able to poke around queues used for fmn internally. For example, they could connect to the queue the workers use and drain the queue. Messages would then get lost.
- Prefetch count/size for queue consumers. This is how many un-acked messages RabbitMQ will push to the SSE server (with size limits). Letting it push more means better performance since there isn't so much back-and-forth chatter acking each message before the next one is sent, but it means more memory consumption on the consumer.

### PR DESCRIPTION
**Pull request by jeremycline**
This adds a blacklist and whitelist for queue names that the SSE server
will serve. It also lets the QoS options for the queue consumer be
configured and the access control header to be set.

fixes #57

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

**Comment by skrzepto** [comment]
lgtm as well

File: fedmsg.d/sse.py [hunk 1]
```diff
@@ -9,6 +9,24 @@
...
+    # whitelist of queues exposed by the SSE server.
+    'fmn.sse.webserver.queue_whitelist': '.+\.id\.fedoraproject\.org$',
+
+    # A regular expression using the standard Python re syntax that defines a
+    # blacklist for queues exposed by the SSE server. Any queue name that is
+    # matched by the regular expression will return a HTTP 403 to the client.
+    #
+    # Note: This is applied _after_ the whitelist so if the queue is matched
+    # by both regular expressions, the queue _will not_ be served.
+    'fmn.sse.webserver.queue_blacklist': None,
```
Comment (skrzepto) [inline fedmsg.d/sse.py:22]: if we have a whitelist isnt a blacklist not necessary? One case where two of them would be useful is we have spammers who pass the whitelist and we need to specifically block their queues.

Comment (jeremycline) [reply fedmsg.d/sse.py:22]: I couldn't think of a particularly good reason to ever use a blacklist, but I thought that someone at some time might want to, and when that time comes, they probably don't want to wait on me to add it. It seemed super simple so I just did it in one pass.

File: tests/test_style.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    os.path.dirname(os.path.join(os.path.dirname(__file__), '../')))
+
+
+class TestStyle(unittest.TestCase):
+    """Run flake8 on the repository directory as part of the unit tests."""
+
+    def test_code_with_flake8(self):
+        """Assert the code is PEP8-compliant"""
+        flake8_command = ['flake8', '--max-line-length', '100', REPO_PATH]
+        self.assertEqual(subprocess.call(flake8_command), 0)
```
Comment (bowlofeggs) [inline tests/test_style.py:33]: Yeah!

**Review by bowlofeggs — APPROVED** [review]

**Pull request merged**